docs/setup/new_environment.md

```diff
@@ -59,7 +59,18 @@ This document will walk you through the process of setting up a new monolith ser
 1. Install Required Packages
 
     ``` bash
-    $ sudo apt update && sudo apt install python-pip sshpass
+    $ sudo apt update && sudo apt install python3-pip sshpass
+    ```
+   Now update your pip3 version; you might encounter installation issues otherwise.
+    ``` bash
+    $ sudo -H pip3 install --upgrade pip
+    ```
+   Make python3 your default python version.
```

Comment:
Why do we need to do this? Doesn't everything run in a virtual env, where you can set the python version? I think changing the default python version can have weird knock-on effects.

Reply:
Running `source /commcare-cloud/control/init.sh` resulted in a package distribution error which seemed to be a compatibility issue I solved by using python 3 rather than 2. I then though it might be good to rather default the _python_ command to python3 rather than 2 for any future compatibility issues, so I just defaulted the system. This was, admittedly, a bold move, but I did not run into any issues so I thought I'd leave it at that.

Also, it may be a good idea since support for python 2 was discontinued in January this year.
(Feel free to let me know if you think this was a...less smart move. I can maybe see if there is a workaround then)

Reply:
We're installing on a fresh Ubuntu 18.04 (so 18.04.5 right now) right? Isn't the default already `python3`?

I think it's only going to be `python2` if you're installing Ubuntu 18.04.1 or upgrading from an older Ubuntu. It might be easier to tell users to download an up-to-date Ubuntu 18.04 release. (18.04.5 Server was released on 2020-08-13.)

... Although ... it might be worth confirming that Python 3 is the default regardless, in case they didn't install a recent 18.04.

Reply:
That's not a bad point. I was assuming the default would not be python3. I'll maybe add a comment for the user to check it first, otherwise "follow these steps to default it to 3"